shell/platform/darwin/macos/framework/Source/FlutterBackingStore.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Cocoa/Cocoa.h>
+#import <Metal/Metal.h>
+
+/**
+ * A marker interface for backing store handles. Typically contain references to the buffers that
```

Comment:
I'm not sure what "marker" means here.

Reply:
Given that this is an interface without any methods, i've called it out as a marker interface: https://en.wikipedia.org/wiki/Marker_interface_pattern.

Reply:
Ah, I see. I wouldn't consider this a marker since it's a clear is-a superclass/subclass relationship, explicitly for the purpose of being a generic type in interfaces. The specific implementations *are* Flutter backing stores; they aren't just random unrelated classes that happen to have backing-store-ness.